microsoft-edge/web-platform/release-notes/136.md

```diff
@@ -182,48 +198,64 @@ See [Fluent Scrollbars Visual Spec](https://docs.google.com/document/d/1haDpb1QI
 <!-- ---------- -->
 ###### AudioContext `interrupted` state
 
-Currently, the Web Audio API doesn't have a mechanism for the browser to interrupt playback for scenarios such as exclusive audio access (VoIP) or when a laptop lid is closed. This change introduces the `interrupted` state to the `AudioContextState` interface. This new state allows the browser to pause playback in the above scenarios and enables web applications to respond appropriately.
+The `AudioContextState` interface now has an `interrupted` state.  The `interrupted` state allows the browser to pause playback in scenarios such as when using exclusive audio access (VoIP) or when a laptop lid is closed, and enables the web app to respond appropriately.
 
 See [AudioContext](https://developer.mozilla.org/docs/Web/API/AudioContext) at MDN.
 
 
 <!-- ---------- -->
-###### Blob URL Partitioning: Fetching/Navigation
+###### Blob URL Partitioning: Fetching/Navigation<!-- todo: section lacks "fetching", try to avoid / and spell out -->
```

Comment:
per corp style, replace '/' with words
why say 'fetching' in heading but not in body?  usually, section body is a superset of heading's words